README.md

```diff
@@ -1,17 +1,17 @@
 [![Build Status](https://dev.azure.com/azure/template-analyzer/_apis/build/status/BuildAndTest?branchName=main)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
 [![Code Coverage](https://shields.io/azure-devops/coverage/azure/template-analyzer/91)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
 
-# Template Best Practice Analyzer (BPA)
+# Template Analyzer (TA)
 
-## What is the Template BPA?
-The Template BPA scans ARM ([Azure Resource Manager](https://docs.microsoft.com/azure/azure-resource-manager/templates/overview)) and [Bicep](https://docs.microsoft.com/azure/azure-resource-manager/bicep/)  Infrastructure-as-Code (IaC) templates to ensure security and best practice checks are being followed before deployment of your Azure solutions.
+## What is Template Analyzer?
+Template Analyzer scans ARM ([Azure Resource Manager](https://docs.microsoft.com/azure/azure-resource-manager/templates/overview)) and [Bicep](https://docs.microsoft.com/azure/azure-resource-manager/bicep/) Infrastructure-as-Code (IaC) templates to ensure security and best practice checks are being followed before deployment of your Azure solutions.
 
-The Template BPA provides a simple and extensible solution to improve the security of your Azure resources before deployment and ensures your templates follow best practices. The Template BPA is designed to be customizable - users can write their own checks and/or enforce only the checks that are relevant for them.
+Template Analyzer provides a simple and extensible solution to improve the security of your Azure resources before deployment and ensures your templates follow best practices. The Template BPA is designed to be customizable - users can write their own checks and/or enforce only the checks that are relevant for them.
 
-## Getting started with the Template BPA
-Download the latest Template BPA release in [the releases section](https://github.com/Azure/template-analyzer/releases).
+## Getting started with Template Analyzer
+Download the latest Template Analyzer release in [the releases section](https://github.com/Azure/template-analyzer/releases).
 
-To preview the rules that come bundled with the Template BPA, explore [the built-in rules](docs/built-in-bpa-rules.md).
+To preview the rules that come bundled with the Template Analyzer, explore [the built-in rules](docs/built-in-bpa-rules.md).
 
 ## Using the Template BPA
 The Template BPA is executed via a command line.  There are two formats to invoke it:
```

Comment:
References here

Reply:
I don't see it

Reply:
I do see it. The file name got you.

Reply:
The header and first sentence, lines 16-17